data-prep/README.md

```diff
@@ -15,9 +15,59 @@ Data Cleaning and Munging
 
 A number of cleaning steps are necessary to use the raw data.
 
-### Project Filtering
+### Consumption Data Cleaning
+
+**Elec Traces**
+
+* Remove duplicates
+  * No duplicates in monthly data
+* Add leading zeroes to `PREM_ID` where missing (length == 8 or 9) in monthly traces.
+
+* 7702 elec monthly traces with unique SA_ID
+
+**Gas Traces**
+
+* Remove duplicates
+  * No duplicates in monthly data
+* Add leading zeroes to `PREM_ID` where missing (length == 8 or 9) in monthly traces.
+
+* 6919 gas monthly traces with unique SA_ID
+
+**Elec Xrefs**
+
+* Add leading zeroes to `char_prem_id` where missing (length == 8 or 9). Appear to have been lost during PGE's export process 
+
+**For Caltrack data analysis: traces with `is_net_metered=Y` should be excluded from analysis.**
```

Comment:
It appears that from monthly billing data we aren't able to identify the premises that are net metered but are not flagged as net metered in the xref table (unless there is something in the rate schedule information that tells us about net metering?  I am not sure what the different rate schedule codes mean).  Consequently my earlier comment doesn't apply if we're talking about monthly billing data.